L83-core-cpp-tls-key-export-feature.md

```diff
@@ -0,0 +1,93 @@
+Title
+----
+* Author(s): Vignesh Babu (Vignesh2208)
+* Approver: ctiller
+* Status: In Review
+* Implemented in: core, cpp
+* Last updated: 07/29/2021
+* Discussion at: https://github.com/grpc/grpc/issues/24944
+
+## Abstract
+
+This proposal discusses additions to the core/cpp public headers to support exporing of TLS Keys on the client and server side to aid in decryption of packet captures using tools such as Wireshark.
+
+## Background
+
+Currently unlike the GRPC Go implementation, GRPC C++ does not allow export of TLS keys at run time. This is needed to decrypt and analyse packet captures for debugging purposes. The keys when exported in files in a specific format, can be read by debugging tools like wireshark and they can automatically decrypt packet captures.
+
+### Related Proposals:
+
+N/A
+
+## Proposal
+
+The proposal extends the existing `grpc::experimental::TlsCredentialsOptions` class with methods to set configuration for supporting Tls Key export at run-time. The proposed changes
```

Comment:
The fact that we're using `experimental` here gives me pause, since experimental APIs don't actually require gRFCs at all.  TlsCreds as a whole is still experimental, and there will be a gRFC for it as a whole when we're ready to de-experimentalize it (see #205 for some work-in-progress).

I suspect this means that we should just add this API to the TlsCreds gRFC that @ZhenLian is working on, rather than having this separate gRFC.  And not having the gRFC merged should not block the implementation, since the API is still currently experimental -- it just means it may be subject to change later.

Reply:
Yes @ZhenLian also thought it would be a good idea to merge them. I'll do that after this is approved.